File: apps/s-2-admin/src/api/apiKey/ApiKeyCreateInput.ts
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { AppModelWhereUniqueInput } from "../appModel/AppModelWhereUniqueInput";

export type ApiKeyCreateInput = {
  note?: string | null;
  expiresAt?: Date | null;
  lastUsedAt?: Date | null;
  hashedKey: string;
  user?: UserWhereUniqueInput | null;
  appField?: AppModelWhereUniqueInput | null;
};

File: apps/cactus-demo-dotnet/src/APIs/ApiKey/Dtos/ApiKeyCreateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class ApiKeyCreateInput
{
    public string? Id { get; set; }

    public string? Note { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime? ExpiresAt { get; set; }

    public DateTime? LastUsedAt { get; set; }

    public string HashedKey { get; set; }

    public User? User { get; set; }

    public AppModel? AppField { get; set; }
}
